doc/tutorials/tutorial_understanding_building_benchmark.rst

```diff
@@ -0,0 +1,312 @@
+.. _Benchopt: https://benchopt.github.io/#
+
+Understanding and building a simple benchmark with Benchopt
+===========================================================
+
+Are you looking for a simple, step-by-step introduction to the `Benchopt`_ library? Then look no further! In this tutorial, we are going to learn how to use the `Benchopt`_ library to compare various algorithms in order to minimize a given cost. This tutorial complements the `write a benchmark <https://benchopt.github.io/how.html>`_ webpage with a more practical, hands-on approach. 
```

Comment:
IMO no need to make Benchopt a link (2 occurrences)

Reply:
I removed the global link definition. I think it is still nice if Benchopt homepage is linked at least once, in case this html is used outside the doc for some reason.